CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 CHANGELOG
 =========
+## 0.35.0 / 2020-03-06
```

Comment:
Let's group `Added` together

Reply:
Sorting by issue type needs to be added to `ddev` then

Reply:
Added here: https://github.com/DataDog/integrations-core/pull/5995